.github/actions/playwright/action.yml

```diff
@@ -14,13 +14,13 @@ inputs:
 runs:
   using: composite
   steps:
-    - uses: actions/checkout@v2
-    - uses: actions/setup-node@v2
+    - uses: actions/checkout@v4
+    - uses: actions/setup-node@v4
       with:
-        node-version: "17"
-    - uses: bahmutov/npm-install@v1
```

Comment:
from what I can tell the main purpose of this action was caching - this is supposed to be builtin into `actions/setup-node` nowadays so hopefully we can just use it to simplify things